worksheet_C/readme.md

```diff
@@ -0,0 +1,19 @@
+## Question A
+Checks a list for duplicate values
+## Question B
+Due to the fact that the list is being checked for duplicates, every pair of values in the lists have to be compared to see if they match, and thus are duplicates, thus why the algorithm is quadratic in it's worst case.
+## Question C
+The algorithm is still correct because the values are still being compared as j is simply comparing the value before i, where i has already compared the values so the algorithm will still run correctly.
```

Comment:
I think you've got the idea (that the new algorithm only checks the ith element against the elements before it in the list, not the ones after it, hence avoiding repeated comparisons). Explanation could be clearer though.